hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardToolDynamoDB.java

```diff
@@ -289,4 +291,30 @@ public void testDestroyUnknownTable() throws Throwable {
         "-meta", "dynamodb://" + getTestTableName(DYNAMODB_TABLE));
   }
 
+  @Test
+  public void testCLIFsckWithoutParam() throws Exception {
+    intercept(ExitUtil.ExitException.class, () -> run(Fsck.NAME));
+  }
+
+  @Test
+  public void testCLIFsckWithParam() throws Exception {
+    final int result = run(S3GuardTool.Fsck.NAME, "-check",
+        "s3a://" + getFileSystem().getBucket());
```

Comment:
this test failed for me during a parallel run. This parallelizable test should have a path which we know is there but is private to this test; we can have another one which invokes on a missing path. The full root scan should be run in the ITestS3GuardDDBRootOperations test, before any cleanup